bliss/simulator/prior.py

```diff
@@ -120,14 +121,17 @@ def sample_prior(self, batch_rcfs) -> TileCatalog:
 
         for rcf in batch_rcfs:
             rcf = tuple(rcf)
-            if rcf in self.rcf_gals_rest:
-                select_gal_rcfs.append(self.rcf_gals_rest[rcf])  # noqa: WPS529
-            else:
-                select_gal_rcfs.append(np.random.choice(list(self.rcf_gals_rest.values())))
-            if rcf in self.rcf_stars_rest:
-                select_star_rcfs.append(self.rcf_stars_rest[rcf])  # noqa: WPS529
-            else:
-                select_star_rcfs.append(np.random.choice(list(self.rcf_stars_rest.values())))
+            # get value if key exists, otherwise pick randomly
+            select_gal_rcfs.append(
+                self.rcf_gals_rest[rcf]
+                if rcf in self.rcf_gals_rest
+                else random.choice(list(self.rcf_gals_rest.values()))
```

Comment:
`numpy.random.choice` apparently doesn't play nice with an iterable of objects (in this case a list of tensors), so we're using `random.choice` instead